packages/blade-svelte/src/components/Button/Button.stories.svelte

```diff
@@ -0,0 +1,103 @@
+<script context="module">
+  import { defineMeta } from '@storybook/addon-svelte-csf';
+  import Button from './Button.svelte';
+  
+  const { Story } = defineMeta({
+    title: 'Button',
+    component: Button,
+    tags: ['autodocs'],
+    args: {
+      children: 'Click me',
+      variant: 'primary',
+      size: 'medium',
+      color: 'primary',
+      isDisabled: false,
+      isLoading: false,
```

Comment:
Button loading state style is broken:

<img width="140" height="80" alt="Image" src="https://github.com/user-attachments/assets/7f0ef5c4-8862-40d1-a6c3-b2e57fbc8175" />